<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageUsers</class>
 <widget class="QWidget" name="PageUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Style Local */
QWidget { 
    font-family: &quot;Segoe UI&quot;, sans-serif; 
    background-color: #f0f2f5; 
}

QFrame.Card { 
    background-color: white; 
    border-radius: 8px; 
    border: 1px solid #e0e0e0; 
}

QLabel {
    color: #333333; 
}

/* --- CHAMPS DE TEXTE --- */
QLineEdit { 
    border: 1px solid #ccc; 
    border-radius: 4px; 
    padding: 6px; 
    background-color: #fafafa; 
    color: #333333; 
}
QLineEdit:focus { 
    border: 1px solid #2d7aed; 
    background-color: white; 
}

/* --- LISTE DÉROULANTE (CORRIGÉ) --- */
QComboBox {
    border: 1px solid #ccc;
    border-radius: 4px;
    padding: 6px;
    background-color: #fafafa;
    color: #333333; /* Texte de la boite fermée */
}
QComboBox:focus {
    border: 1px solid #2d7aed;
    background-color: white;
}
/* Le menu déroulant lui-même */
QComboBox QAbstractItemView {
    background-color: white;
    color: #333333;       /* Texte des options en noir */
    selection-background-color: #2d7aed;
    selection-color: white;
    outline: 0;
}

/* --- BOUTONS --- */
QPushButton.ActionBtn { 
    background-color: #2d7aed; 
    color: white; 
    border-radius: 4px; 
    padding: 8px; 
    font-weight: bold; 
    border: none;
}
QPushButton.ActionBtn:hover {
    background-color: #1a5fc7;
}

/* --- TABLEAU --- */
QTableWidget {
    background-color: white;
    gridline-color: #f0f0f0;
    selection-background-color: #e6f0ff;
    selection-color: #2d7aed;
    color: #333333;
    border: none;
}
QHeaderView::section {
    background-color: #f8f9fa;
    padding: 5px;
    border: none;
    border-bottom: 2px solid #2d7aed;
    font-weight: bold;
    color: #333333;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame_list">
     <property name="class" stdset="0">
      <string>Card</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_list">
      <item>
       <widget class="QLabel" name="label_list_title">
        <property name="text">
         <string>Liste des Utilisateurs</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 16px; font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableUsers">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Nom</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Login</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Rôle</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   
   <item>
    <widget class="QFrame" name="frame_form">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="class" stdset="0">
      <string>Card</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_form">
      <item>
       <widget class="QLabel" name="label_form_title">
        <property name="text">
         <string>Ajouter / Modifier</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 16px; font-weight: bold; margin-bottom: 10px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="placeholderText">
         <string>Nom complet</string>
        </property>
       </widget>
      </item>
      
      <item>
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="placeholderText">
         <string>Login</string>
        </property>
       </widget>
      </item>
      
      <item>
       <widget class="QComboBox" name="combo_role">
        <item>
         <property name="text">
          <string>Médecin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Réceptionniste</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Administrateur</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save_user">
        <property name="class" stdset="0">
         <string>ActionBtn</string>
        </property>
        <property name="text">
         <string>Enregistrer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>